Description: Fix spelling error in binary
Author: Lucas Kanashiro <kanashiro.duarte@gmail.com>
Forwarded: yes, https://github.com/tmate-io/tmate/pull/228
Last-Updated: 2020-11-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tmate-ssh-client.c
+++ b/tmate-ssh-client.c
@@ -449,7 +449,7 @@
 					ssh_get_error(session));
 			return;
 		case SSH_OK:
-			tmate_debug("Session opened, initalizing tmate");
+			tmate_debug("Session opened, initializing tmate");
 			client->state = SSH_BOOTSTRAP;
 		}
 		// fall through
